tools/continuous-ingestor/TestRunbook.ipynb

```diff
@@ -0,0 +1,1207 @@
+    "             FROM {}.{}\n",
+    "            WHERE 1=1\n",
+    "              AND measure_name = 'cpu_user'\n",
+    "            ORDER BY time \n",
+    "            LIMIT 250\"\"\".format(DB_NAME, TABLE_NAME13SNR10)\n",
+    "resultSAW10 = timestream.executeQueryAndReturnAsDataframe(read_client, query, True)\n",
+    "#display.display(resultSAW)\n",
+    "\n",
+    "\n",
+    "plt.rcParams['figure.figsize'] = [20, 15]\n",
+    "fig, ax = plt.subplots(5)\n",
+    "\n",
+    "ax[0].title.set_text('CPU User - Sinus SNR 100 - as Recorded')\n",
+    "ax[0].plot(resultSIN100['time'], resultSIN100['cpu_user'], color='darkorange', \\\n",
+    "           marker='+', markersize=12, mew=2, linewidth=0.5, alpha=0.5)\n",
+    "ax[0].grid(which ='both', axis ='both', linestyle ='--')\n",
+    "\n",
+    "ax[1].title.set_text('CPU User - Sinus SNR 20 - as Recorded')\n",
+    "ax[1].plot(resultSIN20['time'], resultSIN20['cpu_user'], color='darkorange', \\\n",
+    "          marker='+', markersize=12, mew=2, linewidth=0.5, alpha=0.5)\n",
+    "ax[1].grid(which ='both', axis ='both', linestyle ='--')\n",
+    "\n",
+    "ax[2].title.set_text('CPU User - Sinus SNR 1 - as Recorded')\n",
+    "ax[2].plot(resultSIN1['time'], resultSIN1['cpu_user'], color='darkorange', \\\n",
+    "          marker='+', markersize=12, mew=2, linewidth=0.5, alpha=0.5)\n",
+    "ax[2].grid(which ='both', axis ='both', linestyle ='--')\n",
+    "\n",
+    "\n",
+    "ax[3].title.set_text('CPU User - Saw SNR 100 - as Recorded')\n",
+    "ax[3].plot(resultSAW100['time'], resultSAW100['cpu_user'], color='darkgreen', \\\n",
+    "           marker='+', markersize=12, mew=2, linewidth=0.5, alpha=0.5)\n",
+    "ax[3].grid(which ='both', axis ='both', linestyle ='--')\n",
+    "\n",
+    "ax[4].title.set_text('CPU User - Saw SNR 10 - as Recorded')\n",
+    "ax[4].plot(resultSAW10['time'], resultSAW10['cpu_user'], color='darkgreen', \\\n",
+    "           marker='+', markersize=12, mew=2, linewidth=0.5, alpha=0.5)\n",
+    "ax[4].grid(which ='both', axis ='both', linestyle ='--')\n",
+    "ax[4].set_xticklabels('', visilbe=False)\n",
+    "\n",
+    "fig.autofmt_xdate(rotation=60)\n",
+    "fig.set_figwidth(20)\n",
+    "\n",
+    "plt.savefig('sinsawsnr.png')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "instance_type": "ml.t3.medium",
+  "kernelspec": {
+   "display_name": "Python 3 (Data Science)",
+   "language": "python",
+   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:eu-west-1:470317259841:image/datascience-1.0"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
It would be helpful to end this notebook with some optional code to delete the test database we created

Reply:
Added the code to cleanup the database / tables created in this test notebook. 
Anything to add on protecting user from accidently deleting things they dont want to be deleted?